tfhe/src/typed_api/booleans/client_key.rs

```diff
@@ -0,0 +1,25 @@
+use crate::boolean::client_key::ClientKey;
+
+use serde::{Deserialize, Serialize};
+
+use super::{parameters::BooleanParameterSet, FheBoolParameters, types::r#static::StaticBoolParameters};
+
+ #[cfg_attr(doc, cfg(feature = "boolean"))]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct GenericBoolClientKey<P>
```

Comment:
I think we are missing docstrings for the Generic Structs

Reply:
then design/architecture document or top module doc will do

Reply:
tried to add one 
https://github.com/zama-ai/tfhe-rs/blob/0e968f4d2e5835f5a5d09f584d3b1160432bf6cd/tfhe/src/typed_api/design.md